printer_mods/wrez/astrodyne-inlet/README.md

```diff
@@ -1,28 +1,51 @@
 # Astrodyne IEC 320 C14 Power Inlet Housing for V0.1
 
-This mod is a replacement housing for the stock v0.1 power inlet, replacing the [Adam
-Tech
+This mod is a replacement housing for the stock v0.1 power inlet, replacing the
+[Adam Tech
 IEC-GS-1-100](https://www.digikey.com/en/products/detail/adam-tech/IEC-GS-1-100/9831135)
-(which has been on back-order at Digikey for quite a while at the time of
-this writing) with an
-[Astrodyne
-082S/082SM](https://www.mouser.com/ProductDetail/astrodyne-tdi/082sm0150000/?qs=eP2BKZSCXI4KQZVxn1eqpw%3D%3D).
+(which has been on back-order at Digikey for quite a while at the time of this
+writing) with an [Astrodyne/TDI
+082S/082SM](https://www.mouser.com/ProductDetail/Astrodyne-TDI/082S0150000?qs=eP2BKZSCXI7jG836kH3rhw%3D%3D).
+
+In addition to the power inlet you will also need to acquire some inexpensive 5
+mm diameter X 20 mm long 250V fast-acting fuses (either 15A or 10A fuses are
```

Comment:
Please don't use 15A or 20A fuses in a V0.  The sourcing guide shows a 2.5A fuse, and also has a link to one at Digikey.  The 18 gauge wire used for mains in the manual can't even take 20A, and of course the 150W PSU will never draw that much under normal conditions.  I would either remove this whole paragraph, or just change it to say that fuses will need to be purchased and there's a link in the sourcing guide.

Reply:
Yup. Momentary insanity. Changes pushed.